app/robots/vizzy/vizzyStartupWithROS_simulator.xml

```diff
@@ -0,0 +1,62 @@
+<application>
+<name>Vizzy Startup with ROS on gazebo</name>
+    <dependencies>
+      <port>/vizzy_ns</port>
+      <port>/ros</port>
+    </dependencies>
+
+    <module>
+         <name>yarpidl_rosmsg</name>
+          <parameters>--name /typ@/yarpidl</parameters>
+          <node>localhost</node>
+         <tag>yarpidl_rosmsg</tag>
```

Comment:
Hi @plinioMoreno, two small suggestions for the future: (1) you can get rid of the ```<tag>``` parts, they are useless / deprecated; (2) in this XML you have different number of spaces for indentation in different lines, for example before the ```<name>''' parts. It would be clearer for the eye to harmonize it. :)